src/protocols/multi_party_ecdsa/gg_2020/test.rs

```diff
@@ -69,6 +69,47 @@ fn test_sign_n8_t4_ttag6() {
     let _ = sign(4, 8, 6, vec![0, 1, 2, 4, 6, 7], 0, &[0]);
 }
 
+#[test]
+fn test_keygen_orchestration() {
```

Comment:
maybe add a sentence on what this test is doing ?

Reply:
added.